certz/certz.proto

```diff
@@ -452,6 +479,15 @@ message CertificateRevocationListBundle {
   repeated CertificateRevocationList certificate_revocation_lists = 1;
 }
 
+// An authentication policy.
+// This policy defines which Certificate Authority can sign which certificate.
```

Comment:
This line is a bit confusing -> "which Certificate Authority can sign". After pushing the CA bundle, any grpc client trying to connect with grpc-server will be validated against the pushed CA bundle. In what workflow does the signing come into the picture after pushing the CA bundle?

Reply:
That is the whole idea - tell the gRPC server to accept only a subset of certificates that are signed by a particular CA.

Reply:
Got it. It's more like "client cert 'X' is valid if and only if signed by 'X-CA'". If "client-cert 'X' is signed by 'Y-CA'", it is invalid.

Reply:
Correct.